payloads/busybox/Makefile

```diff
@@ -69,7 +69,8 @@ RUNENV_VALIDATE_EXPECTED := Hello world
 RUNENV_TAG := ${COMPONENT}
 
 export define runenv_prep
-	tar -cf - ${PAYLOAD_FILES} | tar -C $(RUNENV_PATH) -xf -
-	mkdir -p $(RUNENV_PATH)/empty_tmp && rm -rf $(RUNENV_PATH)/empty_tmp/* && chmod a+rwx $(RUNENV_PATH)/empty_tmp
+	mkdir -p tmp && rm -rf tmp/* && chmod a+rwx tmp
```

Comment:
do we need to set the owner+group?

Reply:
It becomes root by default when docker builds the image